src/systems/joint_controller/JointController.cc

```diff
@@ -85,23 +90,19 @@ void JointController::Configure(const Entity &_entity,
     return;
   }
 
+  // Ugly, but needed because the sdf::Element::GetElement is not a const
+  // function and _sdf is a const shared pointer to a const sdf::Element.
+  auto ptr = const_cast<sdf::Element *>(_sdf.get());
+
   // Get params from SDF
-  auto jointName = _sdf->Get<std::string>("joint_name");
-  if (jointName.empty())
+  sdf::ElementPtr sdfElem = ptr->GetElement("joint_name");
```

Comment:
Replace this with a call to `GetFirstElement` instead and filter from there. You will not need to change the `const_cast` value.

Reply:
`GetFirstElement` is probably not what you want because it gives you the first child element, not necessarily the one named `joint_name`. As my comment above, a combination of `FindElement` and `GetNextElement` would work best.

Reply:
Got it! Thanks!